src/rshim_fuse.c

```diff
@@ -1002,6 +1003,7 @@ static const struct cuse_methods rshim_misc_fops = {
 typedef struct {
```

Comment:
This change will break existing tools, such as the upstream openocd. 

Can we use a different message? Such as defining rshim_ioctl_msg2, and RSHIM_IOC_READ2, RSHIM_IOC_WRITE2.